.pipelines/templates/template-deploy-int-env.yml

```diff
@@ -0,0 +1,105 @@
+parameters:
+  workingDirectory: ""
+steps:
+  - script: |
+      set -eu
+      cd ${{ parameters.workingDirectory }}
+
+      . secrets/env
+
+      MDM_CERT="$(cat secrets/mdm-cert.pem)"
+      MDM_KEY="$(cat secrets/mdm-key.pem)"
+      SSH_PUBLIC_KEY="${{ parameters.sshPublicKey }}"
+
+      echo "########## Creating RG $RESOURCEGROUP in $LOCATION ##########"
+      az group create -g "$RESOURCEGROUP" -l "$LOCATION" >/dev/null
+      echo "## Done"
+      
+      RP_SPN_ID=$(az group deployment show -n rp-nsg -g $RESOURCEGROUP --query properties.outputs.rpServicePrincipalId.value -o tsv)
+
+      echo "########## Deploying rp-production in RG $RESOURCEGROUP ##########"
+      echo "### domainName            = $DOMAIN_NAME.$PARENT_DOMAIN_NAME"
+      echo "### keyvaultPrefix        = $KEYVAULT_PREFIX"
+      echo "### rpServicePrincipalId  = $RP_SPN_ID"
+      az group deployment create \
```

Comment:
This potentially will change with https://github.com/Azure/ARO-RP/pull/204
Happy to refactor later when mine gets in

Reply:
Is there a possibility to keep all these values as a parameters file somewhere?

Reply:
I would love for this to change some.  I'm not sure what you mean by in a parameters file somewhere. I keep the values to pass in mostly in key vault and link it to the VSO pipeline.

Reply:
I tried to create a parameters file instead of an env file and I ran into issues with the certificates.  I don't need them anymore (now they are references to the certs instead) so I could go back to using a parameter file instead I guess.  I'll still need a few values here so that I can upload the encryption key /certs and update the parent domain. But it should be cleaner on this line.

Reply:
I switched most everything over. Some of the parameters that contained " or were long (I had issues with parameters file being too long, az tool wouldn't load the whole file and there would be a parsing error) I left as environment variables but I passed them in directly instead of through an env file

Reply:
With the change to have 'make secrets' I think we can store these values in the storage account. I'm still thinking about it.

Reply:
I thought about this more and I don't like the idea of storing the parameters file in the storage account behind make secrets because we lose history on how to form it/it's state. I also don't like checking secrets into code. So I don't want to check it in.  So I'm inclined to keep generating it at the time of deploying the rp.